gradle-jdks/build.gradle

```diff
@@ -14,13 +14,16 @@ dependencies {
     api project(':gradle-jdks-distributions')
     api project(':gradle-jdks-json')
     api project(':gradle-jdks-setup')
+    api project(':gradle-jdks-setup-common')
```

Comment:
I think because the files that were moved to `gradle-jdks-setup-common` were originally in `gradle-jdks-distributions`, we should make it an api dep from `gradle-jdks-distributions` not here?